language/libs/oslib.rd

```diff
@@ -108,12 +109,16 @@ items in linux {
 }
 
 items in redhat_family {
-  @cfengine_name="(redhat.!centos.!fedora.!oracle_linux.!scientific_linux.!amazon_linux)"
+  # Ugly but needed as very old rhel do not have the rhel class
+  @cfengine_name="rhel|(redhat.!centos.!fedora.!oracle.!almalinux.!rocky.!scientific_linux.!amazon_linux)"
   rhel,
   centos,
   fedora,
-  oracle_linux,
+  oracle,
+  almalinux,
```

Comment:
shouldn't it be alma ?

Reply:
the class is named almalinux, probably because the name of the distribution is "AlmaLinux" in one word, but we can use alma if we want.